homeassistant/auth/providers/legacy_api_password.py

```diff
@@ -28,7 +28,7 @@
 
 
 def _create_repair_and_validate(config: dict[str, Any]) -> dict[str, Any]:
-    async_create_issue(
+    ir.async_create_issue(
```

Comment:
Tbh even that feels a bit redundant. `async_create_issue` is a pretty recognizable name. Unlikely that any other registry implements a function with the same one.

Maybe we should create a blacklist instead for functions which should require the alias. At first glance that would only be `async_get` and `async_load` although the latter is mainly used in `bootstrap.py`, so it might not even be necessary to change that.

https://github.com/home-assistant/core/blob/87989a88cd4cff6461084434d52f27289e189732/homeassistant/bootstrap.py#L398-L404

Reply:
I have adjusted #118215 accordingly
I will close this PR - and maybe we continue the discussion there?